drivers/sensor/sensor_shell.c

```diff
@@ -57,27 +57,38 @@ const char *sensor_channel_name[SENSOR_CHAN_ALL] = {
 static int handle_channel_by_name(const struct shell *shell, struct device *dev,
 					const char *channel_name)
 {
-	int i;
 	struct sensor_value value[3];
+	char *endptr;
 	int err;
+	int i;
```

Comment:
Are you reverse-chrismas-tree-ing this? :laughing:

Reply:
Always. :)